modules/nodes-cluster-overcommit-project-disable.adoc

```diff
@@ -7,29 +7,61 @@
 = Disabling overcommitment for a project
 
+ifndef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
 When enabled, overcommitment can be disabled per-project. For example, you can allow infrastructure components to be configured independently of overcommitment.
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
 
-.Procedure
+ifdef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
+If required by your use case, you can disable overcommitment on any project that is not managed by Red Hat. For a list of projects that cannot be modified, see _Red Hat Managed resources_ in _Additional resources_.
 
-To disable overcommitment in a project:
+.Prerequisites
+* You are logged in to the cluster using an account with cluster administrator or cluster editor permissions.
 
-ifndef::openshift-rosa,openshift-dedicated[]
-. Create or edit the namespace object file.
-endif::openshift-rosa,openshift-dedicated[]
-// Invalid value: "false": field is immutable, try updating the namespace
-ifdef::openshift-rosa,openshift-dedicated[]
-. Edit the namespace object file.
-endif::openshift-rosa,openshift-dedicated[]
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
 
+.Procedure
+//For OCP and Origin:
+ifndef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
+. Create or edit the namespace object file.
 . Add the following annotation:
 +
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
+//For ROSA, ROSA-HCP, OSD:
+ifdef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
+. Edit the namespace object file.
+.. If you are using the web console:
+... Click *Administration* -> *Namespaces* and click the namespace for the project.
+... Under *Annotations*, click the *Edit* button.
+... Click *Add more* and enter a new annotation using a *Key* of `quota.openshift.io/cluster-resource-override-enabled` and a *Value* of `false`.
+... Click *Save*.
+ifndef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp[]
+.. If you are using the {oc-first}:
+... Edit the namespace:
++
+[source,yaml]
+----
+$ oc edit namespace/<project_name>
+----
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp[]
+ifdef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp[]
+.. If you are using the ROSA CLI (`rosa`):
+... Edit the namespace:
++
+[source,yaml]
+----
+$ rosa edit namespace/<project_name>
+----
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp[]
+... Add the following annotation:
++
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
 [source,yaml]
 ----
 apiVersion: v1
 kind: Namespace
 metadata:
   annotations:
-    quota.openshift.io/cluster-resource-override-enabled: "false" <1>
+    quota.openshift.io/cluster-resource-override-enabled: "false" <.>
```

Comment:
This YAML code is not rendering properly in `openshift-enterprise`

<img width="676" alt="Screenshot 2024-09-26 at 1 59 27 PM" src="https://github.com/user-attachments/assets/d9b38d3d-8b34-4ac7-9ddf-2af4df053ba1">

Reply:
@xenolinux Sorry, I'm not clear on how this isn't rendering correctly?

Reply:
Example screenshot -- the code block is rendering below the number.

<img width="674" alt="Screenshot 2024-09-30 at 4 14 27 PM" src="https://github.com/user-attachments/assets/12859f5d-9352-4285-b925-eb61309761cd">

Reply:
Got more details on Slack - the code block isn't indenting correctly / has become detached from the step

Reply:
Should be fixed in the update

Reply:
Nice!! Thanks.